cypress/e2e/podcasts.cy.js

```diff
@@ -23,6 +23,12 @@ const dragDraftToScript = () => {
     )
   })
 }
+const addDraftSlotItems = () => {
+  cy.get("section[slotno=7] textarea").type("<b>new share item</b>{enter}", {
```

Comment:
@chrisns I've already written this like you said in bold tag and it appears in bold. If you want something else kindly please state because in cypress we cannot find whether  it is WSYWG editor or not.